README.md

```diff
@@ -135,8 +135,17 @@ Note that versions of Node.JS v8 prior to `v8.12.0` will NOT work, because OpenT
 > Client instrumentation for the browser is **experimental** and mostly **unspecified**. If you are interested in
 > helping out, get in touch with the [Client Instrumentation SIG][client-instrumentation-sig].
 
-There is currently no list of officially supported browsers. OpenTelemetry is developed using standard web
-technologies and aims to work in currently supported versions of major browsers.
+Rather that define versions of specific browsers / runtimes, OpenTelemetry sets the minimum supported version based on the
+underlying Language features used.
+
+The current minumum language feature support is set as [ECMAScript 2020](https://www.w3schools.com/jS/js_2020.asp) that are available
```

Comment:
Not sure if w3schools is the most _professional_ place to link to, not expecting something as thick as [the spec](https://tc39.es/ecma262/2020/) but maybe [caniuse](https://caniuse.com/sr_es11) since MDN doesn't have an equivalent page?

Reply:
@dyladan, @pichlermarc Does OpenTelemetry have a specific (official / semi-official) site that we should use for externally referencing language references?

Reply:
@MSNev I'm not aware of any guidelines regarding that - I think we should use a link to the spec as there's no way to misunderstand what we're talking about. If there's a need for additional links we can still add them later on.